### PR TITLE
DOCSP-38814 Add Note About Confirming merge/out NLQ

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Adding a note to the warning, that you now have to confirm execution of merge and out pipelines.
- Also adding the note to the `create-agg-pipeline` page.
- Available starting in version `1.43.0`.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/compass/DOCSP-38814/query-with-natural-language/prompt-natural-language-aggregation/#run-the-aggregation

https://preview-mongodbianfmongodb.gatsbyjs.io/compass/DOCSP-38814/create-agg-pipeline/#run-the-pipeline

## JIRA

https://jira.mongodb.org/browse/DOCSP-38814

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66228cec132d178bc1cfc0d2

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
Hi @alenakhineika could you review this when you have a moment please?

File: source/query-with-natural-language/prompt-natural-language-aggregation.txt [hunk 1]
```diff
@@ -71,7 +71,12 @@ Steps
 
             Aggregation pipelines can contain operators such as 
             :pipeline:`$merge` and :pipeline:`$out` that can 
-            modify your collection's data.
+            modify your collection's data. 
+
+            If the aggregation pipeline generated by natural language 
+            querying contains these operators, you must confirm 
+            execution of the pipeline on the confirmation modal before 
+            the pipeline is executed.
```
Comment (sarah-olson-mongodb) [inline source/query-with-natural-language/prompt-natural-language-aggregation.txt:79]: (minimalism)
This is pretty subjective. But you might tweak a smidge to use a few less words. Spitballing, but something like:
If an AI-generated aggregation pipeline contains these operators, you are prompted for confirmation before the pipeline is deployed.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks a ton @ianf-mongodb. This looks great. I made a minimalism based suggestion. Have a look and let me know what you think. 
LGTM

File: source/query-with-natural-language/prompt-natural-language-aggregation.txt [hunk 2]
```diff
@@ -71,7 +71,12 @@ Steps
 
             Aggregation pipelines can contain operators such as 
             :pipeline:`$merge` and :pipeline:`$out` that can 
-            modify your collection's data.
+            modify your collection's data. 
+
+            If the aggregation pipeline generated by natural language 
+            querying contains these operators, you must confirm
```
Comment (alenakhineika) [inline source/query-with-natural-language/prompt-natural-language-aggregation.txt:77]:
```suggestion
            If the aggregation pipeline contains these operators, you must confirm 
```

We show this warnings for both AI generated and manually entered stages.

**Review by ianf-mongodb — COMMENTED** [review]
Thanks @alenakhineika based on your info, I added the updates to the create aggregation pipeline page as well. This is ready for another look.

**Review by alenakhineika — APPROVED** [review]

File: source/includes/fact-agg-pipeline-merge-out-confirm.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Aggregation pipelines can contain operators such as
```
Comment (mdb-ashley) [inline source/includes/fact-agg-pipeline-merge-out-confirm.rst:1]: [suggestion] This is a little hard to parse. Consider:

Some aggregation operators, like $merge and $out, can modify your collection's data. 

If your aggregation pipeline contains operators that can modify your collection's data, you are prompted for confirmation before the pipeline is executed.

Comment (ianf-mongodb) [reply source/includes/fact-agg-pipeline-merge-out-confirm.rst]: Updated

**Review by mdb-ashley — APPROVED** [review]
LGTM. Thanks, Ian!
